### DISCUSSION TITLE
Authentication

### DISCUSSION THREAD

**Discussion by grantnorwood** [remix-austin/remixaustin-com#68] [Ideas]
Let’s discuss authentication here!

I think we’d like to have a central auth system which is no cost or low cost, and which integrates with the CMS we’d like to implement.  This authentication is for website and content editing permissions, and should be flexible enough to support multiple clients and integrations.

Extra points if it integrates smoothly with GitHub, Google or other popular OAuth providers.

## Potential Solutions 

- [Auth0](https://auth0.com/)
- Firebase

**Comment by brookslybrand** [comment]
Calling bankruptcy on all discussions and issues today. Going to start revamping the direction of this project, so stay tuned 🚀
